Wordle.Api/Wordle.Api/Controllers/LeaderboardController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Wordle.Api.Data;
+using Wordle.Api.Dtos;
+using Wordle.Api.Services;
+
+namespace Wordle.Api.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class LeaderboardController : ControllerBase
+    {
+        private readonly LeaderboardService _leaderboardService;
+
+        public LeaderboardController(LeaderboardService leaderboardService)
+        {
+            _leaderboardService = leaderboardService;
+        }
+
+        [HttpGet("{count}")] // ✅
+        public async Task<ActionResult<List<Player>>> GetTopPlayers(int count)
+        {
+            // Set count to 50 if it's less than 1
+            if (count < 1)
+            {
+                count = 50;
+            }
```

Comment:
I think you should make count a nullable, specify a null case. 
I'm uncomfortable with hardcasting count to 50, what if there are less than 50 players in total?
I think it would be better if you asked the db how many valid (for top scores) players exist in the database, and use that number as the min instead. You may need to go to your Leaderboard.vue and have some message if the leaderboard is empty